datafusion/physical-expr-common/src/sort_expr.rs

```diff
@@ -30,6 +30,51 @@ use datafusion_common::Result;
 use datafusion_expr_common::columnar_value::ColumnarValue;
 
 /// Represents Sort operation for a column in a RecordBatch
+///
+/// Example:
```

Comment:
This looks like a massive example but it renders neatly I think:

![Screenshot 2024-09-23 at 8 56 58 AM](https://github.com/user-attachments/assets/fd982ed2-5f62-400a-b89d-166d922a6c54)